### PR TITLE
Fixes #8733: Add a test for maximum title level

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8733

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: tests/check_title_syntax.sh [hunk 1]
```diff
@@ -19,3 +19,24 @@ if [ ${ERROR} -ne 0 ]; then
...
     echo "File ending by newlines: OK"
 fi
+
+# Test that there is not more than 5 "=" in title levels
+
+ERROR=0
+
+for entry in $(find $(dirname $0)/../*_* -name '*txt')
+do
+    LINE=$(grep -E "^======+ " ${entry})
```
Comment (peckpeck) [inline tests/check_title_syntax.sh]: You said "Asciidoc supports 6 "=" for title levels, we should test this to avoid breaking the doc."
A line with 6 = will match here.

Comment (amousset) [inline tests/check_title_syntax.sh]: The maximum is 5, I fixed the issue description.
